lib/compress/zstd_ldm.h

```diff
@@ -28,18 +28,18 @@ extern "C" {
  * ZSTD_ldm_generateSequences():
  *
  * Generates the sequences using the long distance match finder.
- * The sequences completely parse a prefix of the source, but leave off the last
- * literals. Returns the number of sequences generated into `sequences`. The
- * user must have called ZSTD_window_update() for all of the input they have,
- * even if they pass it to ZSTD_ldm_generateSequences() in chunks.
+ * Generates long range matching sequences in `sequences`, which parse a prefix
+ * of the source. `nbSeq` must be large enough to store every sequence.
```

Comment:
note 1 : `nbSeq` is actually the size / capacity of `sequences`.

note 2 : 
> `nbSeq` must be large enough to store every sequence.

is there a way to ensure this condition beforehand ?

Reply:
Yeah, there is `ZSTD_ldm_getMaxNbSeq()`.